bears/documentation/DocumentationStyleBear.py

```diff
@@ -0,0 +1,87 @@
+from coalib.bearlib.languages.documentation.DocumentationComment import (
+    DocumentationComment)
+from coalib.bearlib.languages.documentation.DocstyleDefinition import (
+    DocstyleDefinition)
+from coalib.bearlib.languages.documentation.DocumentationExtraction import (
```

Comment:
you don't need to modify the `__init__` file for that :O
just put @AbdealiJK 's line of code at the top of the file^^